airflow/jobs.py

```diff
@@ -191,11 +198,16 @@ def run(self):
         self.id = id_
 
         # Run
-        self._execute()
+        try:
```

Comment:
This try catch is unnecessary; by raising an `AirflowException`, you will automatically trigger a failure/retry.